### ISSUE TITLE
The MLMACC without only MLM

### ISSUE THREAD

**Issue by YueLiao** [jackroos/VL-BERT#25]
Thanks for your great work! 
Could you provide the MLMACC, including "val" and "train", on the pretraining stage based on the base-bert model and only conceptual caption dataset? 

Additionally, it is easy to be stuck during the pretraining process due to the large I-O. I wonder if you could provide the specific config based on the "slurm setting", like using mc or not, the num_workers per GPU.

The training loss suddenly increases to 'nan' after training 2 epochs. Any methods to avoid this?

**Comment by jackroos** [comment]
@YueLiao 
1. If you meet 'nan' in training, you can resume from last checkpoint that the loss is normal.
2. IO issue may be related to your slurm cluster.
3. For the pre-training accuracy, I would try to find the logs and reply you if I could find them.
Thanks!

**Comment by YueLiao** [comment]
Thanks for your reply. 
1. Got it.
2. I have solved it by using lmdb.
